kubernetes-json-schema/v1.17.0-standalone-strict/analyzer-troubleshoot-v1beta1.json

```diff
@@ -32,7 +32,6 @@
                     "type": "string"
                   },
                   "exclude": {
-                    "description": "BoolOrString is a type that can hold an bool or a string.  When used in JSON or YAML marshalling and unmarshalling, it produces or consumes the inner type.  This allows you to have, for example, a JSON field that can accept a booolean string or raw bool.",
```

Comment:
idk why this got removed

Reply:
yeah, i had this too.  some backwards compatibility with generator